### ISSUE TITLE
How does the ROUNDTRIP operator in gie work?

### ISSUE THREAD

**Issue by kbevers** [OSGeo/PROJ#617]
There is a `ROUNDTRIP` operation in gie, but it is not documented. I need it, but can't figure out how it works.

I would expect something like 
```
operation   +proj=aea
tolerance   0.0001
roundtrip   10 50
```

to work, but it never registers the test input:
```
-------------------------------------------------------------------------------
Reading file 'C:\dev\proj4\test\gie\test.gie'
-------------------------------------------------------------------------------
total:  0 tests succeeded,   0 tests failed.
-------------------------------------------------------------------------------
```

**Comment by busstoptaktik** [comment]
There was an `else` block missing, for updating when things went OK. If your test had failed, `gie` would have complained. Resolved in commit 95d51f1df697d0ab3367ee92af40c1494878ca92

**Comment by kbevers** [comment]
Thanks.

Just for the record, the proper syntax is
```
operation   +proj=aea
accept      10 50
roundtrip   1000
```
